modules/data.land/R/extract_soil_nc.R

```diff
@@ -1,3 +1,63 @@
+#' Extract soil data from gssurgo
+#'
+#' @param outdir 
+#' @param lat 
+#' @param lon 
+#'
+#' @return
```

Comment:
Please fill these in, or use `@inheritParams` for the `@params`. I know @infotroph has been aggressively pushing (and rightly so) for more documentation of return types.